frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-vehicle-count/radiogram-card-content-vehicle-count.component.html

```diff
@@ -0,0 +1,27 @@
+<h5 class="card-title">Meldung über Fahrzeuganzahlen</h5>
+
+<p class="card-text" *ngIf="totalVehicleCount === 0">
+    Derzeit befinden sich im Bereich {{ simulatedRegionName$ | async }} keine
+    Fahrzeuge.
+</p>
+
+<p class="card-text" *ngIf="totalVehicleCount > 0">
+    Derzeit befinden sich im Bereich {{ simulatedRegionName$ | async }}
+    {{ totalVehicleCount }} Fahrzeuge, die unter die folgenden Kategorien
+    fallen:
+</p>
+
+<table *ngIf="totalVehicleCount > 0" class="table card-text">
+    <thead>
+        <tr>
+            <th class="w-50">Kategorie</th>
+            <th class="w-50">Anzahl</th>
```

Comment:
Since the name of vehicle types will probably be longer that the decimal number, I suggest using `w-75` and `w-25` instead. This would also be consistent with the other PRs

Reply:
Done